src/utils/logic-app/designerUtils.ts

```diff
@@ -455,7 +455,6 @@ export function getWebviewContentForDesigner({ authorization, callbacks, definit
                                 SHOW_TOKENS_FOR_FOREACH: !!options.featureFlags && !!options.featureFlags.showforeachtokens,
                                 SHOW_TRIGGER_RECURRENCE: true,
                                 SHOW_VARIABLE_ACTIONS: true,
-                                SUPPORT_INTEGRATION_SERVICE_ENVIRONMENT: iseSupported,
```

Comment:
Is this no longer needed?

Reply:
Yes, Lakshmi removed this feature flag in this week's release.